src/ace.F90

```diff
@@ -378,8 +379,8 @@ subroutine read_ace_table(i_table, i_listing)
       if (data_0K) then
         continue
       else
-        call read_nu_data(nuc)
         call read_reactions(nuc)
+        call read_nu_data(nuc)
```

Comment:
Ditto here -- I think I'll do a separate PR moving around routines to their proper places if you're ok with it.